### ISSUE TITLE
FHIR resource for karnofsky

### ISSUE THREAD

**Issue by joostholslag** [RSO-Zuid-Limburg/RSO-ZL-clinical-models#3]
@gasperr currently the KPS maps to questionnaire response, I think ‘observation’ would be a better fit. Would you agree? What do you think about the impact?

**Comment by gasperr** [comment]
I agree. FHIR Connect mappings at the moment are written based on what Vitaly produces. But overall it's not aligned with whatever FHIR IG would be the proper corresponding model to that openEHR template. 

We can work on finding out the right FHIR IG for this (or is there already one?) and we write fhir connect mappings for that.

**Comment by joostholslag** [comment]
Agreed. I don't know of a FHIR IG for KPS. There's no zib/nl-core afaik. Though there might be one for the project Nedap needed the KPS for. I could check. 
Did you give any thought to generating a fhir profile from the openEHR KPS template?
